slack_bolt/adapter/wsgi/http_request.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Dict
+
+from .internals import ENCODING
+
+
+class WsgiHttpRequest:
+    """This Class uses the PEP 3333 standard to extract request information
+    from the WSGI web server running the application
+
+    PEP 3333: https://peps.python.org/pep-3333/
+    """
+
+    __slots__ = ("method", "path", "query_string", "protocol", "environ")
+
+    def __init__(self, environ: Dict[str, Any]):
+        self.method: str = environ.get("REQUEST_METHOD", "GET")
+        self.path: str = environ.get("PATH_INFO", "")
+        self.query_string: str = environ.get("QUERY_STRING", "")
+        self.protocol: str = environ.get("SERVER_PROTOCOL", "")
+        self.environ = environ
+
+    def get_headers(self) -> Dict[str, str]:
+        headers = {}
+        for key, value in self.environ.items():
+            if key in {"CONTENT_LENGTH", "CONTENT_TYPE"}:
+                name = key.lower().replace("_", "-")
+                headers[name] = value
+            if key.startswith("HTTP_"):
+                name = key[len("HTTP_"):].lower().replace("_", "-")  # fmt: skip
```

Comment:
Turns out the black formatter was cause the Flake8 [issue](https://github.com/psf/black/issues/1323), the workaround if to add `# fmt: skip` on the line causing the issue

Reply:
The Black Formatter will add a whitespace between `len("HTTP_")` and `:`, Flake8 does not align with this syntax, the Black Formatter does not allow ignoring specific rules

`key[len("HTTP_") :].lower().replace("_", "-")`

Reply:
this makes sense; thanks for clarifying it